charts/ingress-nginx/values.yaml

```diff
@@ -1074,6 +1074,7 @@ defaultBackend:
   priorityClassName: ""
   # -- Labels to be added to the default backend resources
   labels: {}
+  topologySpreadConstraints: []
```

Comment:
Could you please take a look at how we are doing that for the controller? There already are values for affinity and tolerations, so maybe move that key up right below `defaultBackend.affinity` and add some docs and examples like that, especially because it's a templatable value:

```suggestion
  # -- Topology spread constraints rely on node labels to identify the topology domain(s) that each Node is in.
  # Ref.: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
  topologySpreadConstraints: []
  # - labelSelector:
  #     matchLabels:
  #       app.kubernetes.io/name: '{{ include "ingress-nginx.name" . }}'
  #       app.kubernetes.io/instance: '{{ .Release.Name }}'
  #       app.kubernetes.io/component: default-backend
  #   topologyKey: topology.kubernetes.io/zone
  #   maxSkew: 1
  #   whenUnsatisfiable: ScheduleAnyway
  # - labelSelector:
  #     matchLabels:
  #       app.kubernetes.io/name: '{{ include "ingress-nginx.name" . }}'
  #       app.kubernetes.io/instance: '{{ .Release.Name }}'
  #       app.kubernetes.io/component: default-backend
  #   topologyKey: kubernetes.io/hostname
  #   maxSkew: 1
  #   whenUnsatisfiable: ScheduleAnyway
```